Unix/engine/engine.c

```diff
@@ -0,0 +1,119 @@
+/*
+**==============================================================================
+**
+** Copyright (c) Microsoft Corporation. All rights reserved. See file LICENSE
+** for license information.
+**
+**==============================================================================
+*/
+
+#include <server/server.h>
+
+static Options s_opts;
+static ServerData s_data;
+
+int enginemain(int argc, const char* argv[])
```

Comment:
No common header file, and this wasn't extern "C"'d.  What happens if `Unix/engine/enginemaincxx.cpp` is used (I guess on HP-UX, for whatever reason)?

I'm guessing you didn't test this on that platform? Does it even pass `PBUILD`?